### PR TITLE
Add zach's post on reasons to upgrade, add zach to yml file, add Fion…

### PR DESCRIPTION
**Pull request by FionaDL**
Hey, here is @zgoldie blog post. We still need to add a link on performance as well as Zach's personal info in the config file. Since I will be out the next few days I at least wanted to get the PR started.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -97,9 +97,16 @@ authors:
...
     display_name: "Noah Gibbs"
     gravatar: 5e8107f48d4471a40de325151d589b6d
     twitter: codefolio
     github: noahgibbs
-    bio: "An old programmer who moved from the US to Scotland. Noah writes about Ruby and Rails performance in many places. Also wrote Rebuilding Rails and Mastering Software Technique."
\ No newline at end of file
+    bio: "An old programmer who moved from the US to Scotland. Noah writes about Ruby and Rails performance in many places. Also wrote Rebuilding Rails and Mastering Software Technique."
+  zach:
+    display_name: "Zach Goldie"
+    gravatar: 5e8107f48d4471a40de325151d589b6d
```
Comment (lubc) [inline _config.yml:109]: Just a reminder to update this

File: _posts/2020-11-24-how-to-pitch-an-upgrade-to-your-boss.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+And if you do hire them, they will be keeping an eye out for other jobs where they won’t have their free time interrupted with emergencies that shouldn’t have required their input.
+
+So if you are having issues with hiring and retaining developers, upgrading Rails will remove a big stumbling block.
+
+## New features with each version
+
+New Rails versions come with a set of new features that applications can be built with.
+
+Some will bring obvious benefits, such as Action Text providing text formatting controls for your users. Others can help in the background, such as Active Record which allows an application to connect to multiple databases.
```
[the remark this answers is not in the document]
Comment (FionaDL) [reply _posts/2020-11-24-how-to-pitch-an-upgrade-to-your-boss.md:51]: @lubc will you accept this commit if you think it sounds okay?

[the remark this answers is not in the document]
Comment (lubc) [reply _posts/2020-11-24-how-to-pitch-an-upgrade-to-your-boss.md:51]: @FionaDL looks good 👍 

PS. how did you edit my comment :squirrel:

**Comment by FionaDL** [comment]
@lubc I think this post should be good to go :)

**Review by lubc — APPROVED** [review]
@FionaDL Will publish this in the morning 👍

**Pull request merged**